dvc/tree/repo.py

```diff
@@ -403,29 +403,17 @@ def get_file_hash(self, path_info):
                 pass
         return HashInfo(self.PARAM_CHECKSUM, file_md5(path_info, self)[0])
 
-    def copytree(self, top, dest):
```

Comment:
This has always been `download`, but we didn't see it before the tree generalization.